pages/cloudflare/index.mdx

```diff
@@ -44,15 +45,13 @@ To help improve compatibility, we encourage you to [report bugs](https://github.
 - [x] [Dynamic routes](https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes)
 - [x] [Static Site Generation (SSG)](https://nextjs.org/docs/app/building-your-application/rendering/server-components#static-rendering-default)
 - [x] [Server-Side Rendering (SSR)](https://nextjs.org/docs/app/building-your-application/rendering/server-components)
-- [x] [Incremental Static Regeneration<sup>1</sup> (ISR)](https://nextjs.org/docs/app/building-your-application/data-fetching/incremental-static-regeneration)
-
-<sup>1</sup> "Manual" revalidation is not supported (i.e. [`revalidateTag()`](https://nextjs.org/docs/app/api-reference/functions/revalidateTag) or [`revalidatePath()`](https://nextjs.org/docs/app/api-reference/functions/revalidatePath))
```

Comment:
> ISR routes do not currently seem to support either manual or time-based revalidation.

That's what l49 says

Reply:
Line 49 only says manual revalidation isn't supported. It doesn't say time-based (the primary way of using ISR) isn't supported, suggesting that it is which is why it's in the supported list. If neither manual or time-based are supported, then the feature isn't supported. When I was testing some example apps over the past few times, time-based was not currently working - at the moment, it seems to have the same experience as ISR routes on next-on-pages from what i can tell.